### ISSUE TITLE
After export,emoji doesn't render as in web app

### ISSUE THREAD

**Issue by trya2l** [typst/typst#4338]
### Description

Hi guys,

I've tried to add some emojis in my document using the `#emoji` syntax and copy-pasting some from gnome characters app (the flags).

Although they are correctly printed in the web app, they do not appear the same way after export.

![screenshot-24-06-05-19-28-48](https://github.com/typst/typst/assets/79646756/f7f3bd49-dd5c-4d8f-bdbf-f38b5d820833)
In the web app

![screenshot-24-06-05-19-29-01](https://github.com/typst/typst/assets/79646756/e2f2509d-16b7-4b3b-8783-35ba09ff7b2c) 
After export

Using typst cli, they won't show up at all.

![screenshot-24-06-05-19-29-25](https://github.com/typst/typst/assets/79646756/c90cc9c7-0f4e-493d-b574-d2e9f22b0902)

Evince print this message when opening the pdf:

```sh
some font thing failed
some font thing failed
some font thing failed
some font thing failed
some font thing failed
...
```

Here is the input in the web app editor

![screenshot-24-06-05-19-34-35](https://github.com/typst/typst/assets/79646756/24d3d786-9c13-4c0c-98f3-b94a60ff1e37)

and in neovim

![screenshot-24-06-05-19-35-15](https://github.com/typst/typst/assets/79646756/f5442222-3b66-42f3-80f0-ae97fdb521b7)

Thanks.

NB: I'm using the `Roboto` font

### Operating system

Web app

### Typst version

- [X] I am using the latest version of Typst

**Comment by Enivex** [comment]
https://github.com/typst/typst/issues/144

This is fixed on main, and will appear in version 0.12

**Comment by trya2l** [comment]
Cool! Thanks
